src/pages/QuestionPage.tsx

```diff
@@ -67,6 +67,7 @@ const QuestionPage: FC = () => {
                               onAnswerUpdate: handlerFactory(questionID, partID, sectionID, taskID),
                               ...instanceToPlain(task),
                               answer: parseAnswer(answer, task.type),
+                              questionText: section.instructions,
```

Comment:
We want to show section.instructions in the modal with the handwriting canvas. We are unsure how else to access this if not passed down through TaskFactory

Reply:
DRP prompts me to ask "why"? 
I would personally stick to a solution _without_ that and then at some point down the line we can always implement it if the user feedback points in that direction.

Reply:
Counter: as users, this is something we want.

We want the ability to see the question we are answering in the handwriting modal pop-up - this is because when you're writing an answer you keep referring to the question, and this is something we found as soon as we tried to test it. 
Otherwise, you have to keep exiting the modal every time to re-read the question and that's not particularly convenient...

Reply:
Mhmh. I appreciate that but this solution doesn't quite solve the problem. There's a very inconsistent use of question instructions, part instructions, section instructions and task instructions. What I mean is that there's often no guarantee that the relevant context to the canvas is in the section's instructions. The best way around it could be to place the canvas on the screen in a way that doesn't conceal the underlying question. This is why I'd say:
(a) let's stick to the clean props model we have for the moment, keeping things as they are in the video
(b) let's think about how to tackle this issue in a more elegant way once the basics are all in